api/s2n.h

```diff
@@ -178,6 +178,8 @@ extern struct s2n_cert_chain_and_key *s2n_cert_chain_and_key_new(void);
 S2N_API
 extern int s2n_cert_chain_and_key_load_pem(struct s2n_cert_chain_and_key *chain_and_key, const char *chain_pem, const char *private_key_pem);
 S2N_API
+extern int s2n_cert_chain_and_key_load_pem_using_len(struct s2n_cert_chain_and_key *chain_and_key, char *chain_pem, uint32_t chain_len, char *private_key_pem, uint32_t private_key_len);
```

Comment:
`size_t` would be more appropriate than `uint32_t`. However it seems like the API already makes use of both, so there isn't an obvious right or wrong anymore. Does the stuffer only allow `uint32_t` so that we prefer that?

Reply:
The members of blob, stuffer and it's related functions are of type `uint32_t`. So that's why decided to stick with `uint32_t`.